<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDataListWidget</class>
 <widget class="QWidget" name="TestDataListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="vwidget" native="true">
       <property name="minimumSize">
        <size>
         <width>1000</width>
         <height>0</height>
        </size>
       </property>
       <property name="widgetStyle" stdset="0">
        <string>BGStyle</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="topMargin">
         <number>65</number>
        </property>
        <property name="bottomMargin">
         <number>155</number>
        </property>
        <item>
         <widget class="QWidget" name="hWRunInfo1" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
           <item>
            <widget class="QLabel" name="label">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>36</pointsize>
              </font>
             </property>
             <property name="text">
              <string>①</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayPaperPos">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>15</number>
             </property>
             <property name="bottomMargin">
              <number>15</number>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="hWRunInfo122" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>65</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>65</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>36</pointsize>
              </font>
             </property>
             <property name="text">
              <string>②</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="hLayPaperPos1">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>8</number>
             </property>
             <property name="rightMargin">
              <number>15</number>
             </property>
             <property name="bottomMargin">
              <number>15</number>
             </property>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
